src/firefly/js/tables/ui/BasicTable.jsx

```diff
@@ -30,21 +28,18 @@ export class BasicTable extends React.Component {
             columnWidths: makeColWidth(props.columns, props.data, props.showUnits)
         };
 
-        this.onResize = this.onResize.bind(this);
+        this.onResize =  debounce((size) => {
+                if (size) {
+                    var widthPx = size.width;
+                    var heightPx = size.height;
+                    this.setState({widthPx, heightPx});
+                }
+            }, 100);
```

Comment:
If i use a different URL table than the default, such as http://web.ipac.caltech.edu/staff/roby/demo/test-table4.tbl, the table appears with column header but without rows, all background white. This happen also in 'dev' currently.